fastlane/lib/fastlane/actions/git_add.rb

```diff
@@ -34,7 +44,12 @@ def self.available_options
                                              UI.user_error!("Couldn't find file at path '#{x}'") unless File.exist?(x)
                                            end
                                          end
-                                       end)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :pathspec,
+                                       description: "The pathspec you want to add files from",
+                                       is_string: true,
```

Comment:
I think it might be better this to `false`, so we can add an array of pathspecs like:
```ruby
git_add(pathspec: ["*.txt", "*.md"])
```